src/pal/src/misc/cgroup.cpp

```diff
@@ -211,11 +345,15 @@ class CGroup
 
             if (strncmp(filesystemType, "cgroup", 6) == 0)
             {
+                if (strncmp(filesystemType, "cgroup2", 8) == 0)
+                {
+                    *is_cgroupv2 = true;
+                }
```

Comment:
Maybe make this assignment unconditional, so `is_cgroupv2` is also assigned when filesystemType is "cgroup":
`*is_cgroupv2 = strcmp(filesystemType, "cgroup2") == 0;`